tests/robustness/model/deterministic.go

```diff
@@ -75,6 +76,15 @@ func (s EtcdState) apply(request EtcdRequest, response EtcdResponse) (bool, Etcd
 	return Match(MaybeEtcdResponse{EtcdResponse: response}, modelResponse), newState
 }
 
+func (s EtcdState) clone() EtcdState {
```

Comment:
I have seen similar methods to this called `DeepCopy`. At least such naming scheme was used in Kubernetes code.